cmd/exec/exec.go

```diff
@@ -2,6 +2,7 @@ package exec
 
 import (
 	"fmt"
+	"path/filepath"
```

Comment:
The idea is - abstract the file path to io package- no other package should know where the file is or what file to read from.

Wdyt?